src/parsing.jl

```diff
@@ -234,7 +234,7 @@ Same as `parse(T, s, rounding_mode)`, but also accept string representing ration
 """
 function parse_num(T, s, rounding_mode)
     if '/' in s
-        num, denum = parse.(BigInt, split(s, '/'))
+        num, denum = parse.(BigInt, split(s, '/'; keepempty = false))
         return T(num//denum, rounding_mode)
     end
     return T(parse(BigFloat, s), rounding_mode)
```

Comment:
As you mentioned, this is probably worth a new PR. Yet, here there are some comments: Since `T` is a type, shouldn't we explicit that? I mean, is it referring to an Interval, or a numberic (<: AbstractFloat) type? I think the case of rational numbers / rational intervals should be handled separately...

Reply:
As far as I can tell, in the current implementation, `T` will always be an `AbstractFloat` even for `I"1//2"`.
Eventually, yes we will need to have a different method for `Rational` bounds.